common/retry/retrypolicy.go

```diff
@@ -18,7 +18,7 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package backoff
+package retry
```

Comment:
If the package name is "retry", may be consider rename RetryPolicy -> Policy.

Reply:
according to the agreement, rename to backoff